File: sub-mysql-web-mapper/src/main/java/org/github/base/service/impl/SysMenuServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.SysMenuEntity;
import org.github.base.mapper.ISysMenuMapper;
import org.github.base.service.ISysMenuService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 菜单管理 服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-21
 */
@Service
public class SysMenuServiceImpl extends ServiceImpl<ISysMenuMapper, SysMenuEntity> implements ISysMenuService {}

File: sub-mysql-mydb-service/src/main/java/org/github/mysql/mydb/base/service/impl/SysMenuServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.mydb.base.service.impl;

import org.github.mysql.mydb.base.entity.SysMenuEntity;
import org.github.mysql.mydb.base.mapper.ISysMenuMapper;
import org.github.mysql.mydb.base.service.ISysMenuService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 菜单表 服务实现类
 *
 * @author JYD_XL
 */
@Service
open class SysMenuServiceImpl : ServiceImpl<ISysMenuMapper, SysMenuEntity>(), ISysMenuService
